module7/assignment6.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>First Mobile Example</title>
+  <meta charset="utf-8">
+  <link rel="apple-touch-icon" href="icons/touch-icon2.png">
+  <link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css" />
+  <script src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+  <script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+  <link rel="stylesheet" href="_css/themes/theme1.css">
```

Comment:
looks like you tried to include an icon for ios - but not a sized icon and not a tab icon. check out the <icon> tag for the tab icon.